web-components/src/components/table-advanced/TableAdvanced.ts

```diff
@@ -58,6 +59,7 @@ export namespace TableAdvanced {
 
     private dragover = (e: Event) => e.preventDefault();
 
+    @queryAll("tr[tabindex]") tableRaws?: HTMLTableRowElement[];
```

Comment:
spelling: did you mean `tableRows`?

Reply:
Yes, thanks! It's my mistake. Will fix it.